### PR TITLE
remove reminders from dialog engine

### PR DESCRIPTION
**Pull request by Pastromhaug**
@jeffreysilver anything come to mind that I could be missing here?

### CODE REVIEW

File: stopcovid/dialog/models/state.py [hunk 1]
```diff
@@ -83,7 +83,6 @@ class PromptStateSchema(Schema):
     slug = fields.Str(required=True)
     start_time = fields.DateTime(required=True)
     failures = fields.Int(allow_none=True)
-    reminder_triggered = fields.Boolean(allow_none=True)
```
Comment (jeffreysilver) [inline stopcovid/dialog/models/state.py:86]: Are we sure that removing this from the schema isn't going to cause errors for dicts that are already in dynamo?

Comment (Pastromhaug) [reply stopcovid/dialog/models/state.py:86]: I think so, but good call I'll double check that

Comment (Pastromhaug) [reply stopcovid/dialog/models/state.py:86]: very good catch actually, marshmallow blows up when you pass it extra fields. I'll add this back

Comment (jeffreysilver) [reply stopcovid/dialog/models/state.py:86]: 👍

**Pull request merged**